query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/filters/where_unique.rs

```diff
@@ -11,7 +11,7 @@ mod where_unique {
             &runner,
             "query { findUniqueUser(where: {}){ id }}",
             2009,
-            "Expected exactly one field to be present, got 0."
+            "Expected a minimum of 1 fields of (id, email, first_name_last_name) to be present, got 0."
```

Comment:
The "extended where" is a temporary name that we use to qualify the feature we're building. But those tests are for a where unique. Once we remove the feature flag, we'll refer to this as where unique again. Why do you think we should rename the file?

Reply:
I like the idea of extended where because these sets of tests are around that concept. yes its just normal where but if I came to the code base and looked at a `where_unique` test I would think its a little strange to be there. But if we have the `extended_where_unique` it gives me context. There is something related to a new feature that was added later to the `where_unique` functionality

Reply:
> but if I came to the code base and looked at a where_unique test I would think its a little strange to be there

I'm sorry but I don't understand why you'd find that strange. This `where_unique.rs` test file already existed. All I did was add a couple more tests to it. In 2 months, we'll GA and remove this feature flag and extended_where_unique won't make any sense anymore. It won't be easier either from a visibility/searchability perspective because we'll have where unique tests spread in two different files for no obvious reasons because extendedWhereUnique won't be a concept anymore